LocalMultiplayerAgent/SettingsJsonGenerator/main.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<body onload="onInputChange()">
+<script id="myScript" src="scripts.js"></script>
+
+<table><tr><td>
+	
+<p>
+<h1>LocalMultiplayerAgent Json inputs</h1>
+<h4>PlayFab Configuration</h4>
+TitleId: <input onchange="onInputChange()" type="text" id="TitleId" value="">
+<br />
+Region: <!--input type="text" id="Region" value="WestUs"-->
+<select onchange="onInputChange()" name="Region" id="Region">
+	<option value="EastUs">EastUs</option>
+	<option value="EastUs2">EastUs2</option>
+	<option value="WestUs">WestUs</option>
+	<option value="WestUs2">WestUs2</option>
+	<option value="NorthCentralUs">NorthCentralUs</option>
+	<option value="SouthCentralUs">SouthCentralUs</option>
+	<option value="CentralUs">CentralUs</option>
+	<option value="WestCentralUs">WestCentralUs</option>
+	<option value="JapanEast">JapanEast</option>
+	<option value="JapanWest">JapanWest</option>
+	<option value="WestEurope">WestEurope</option>
+	<option value="NorthEurope">NorthEurope</option>
+	<option value="BrazilSouth">BrazilSouth</option>
+	<option value="AustrailiaEast">AustrailiaEast</option>
+	<option value="AustrailiaSoutheast">AustrailiaSoutheast</option>
+	<option value="EastAsia">EastAsia</option>
+	<option value="SoutheastAsia">SoutheastAsia</option>
+	<option value="SouthAfricaNorth">SouthAfricaNorth</option>
+	<option value="UAENorth">UAENorth</option>
+	<option value="CentralIndia">CentralIndia</option>
+	<option value="FranceCentral">FranceCentral</option>
+	<option value="KoreaCentral">KoreaCentral</option>
+</select>
+
+<br />
+Initial Players: <input onchange="onInputChange()" type="text" size="100%" id="InitialPlayers" value="Player1, Player2">
```

Comment:
Will this work with more players?

Reply:
We can have a tooltip that mentions that player names must be comma separated I guess

Reply:
Tooltips will be a lot more work, so I haven't invested that time yet.
Otherwise yes, tested as working with any number of players